### PR TITLE
Incident response guide rough draft

### PR DESCRIPTION
**Pull request by metadave**
cc @jgmize @bensternthal @pmac @jwhitlock @escattone

### CODE REVIEW

File: docs/incident_response.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+Stay cool, take a deep breath. We'll solve this problem together, and this guide is here to help.
+
+This document does not describe the on-call or escalation process, but rather the process we use once an issue has been identified.
+
+## Declaring an incident and the incident leader (IL) role
+
+The incident leader (IL) serves as the primary point of contact and coordinator for an incident. This person may have been called by the moc, or just noticed that a service is broken. 
+
+When an incident occurs, the IL posts messages in IRC stating:
```
Comment (bensternthal) [inline docs/incident_response.md]: Which IRC channel is this information posted to?

File: docs/incident_response.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+> don't post phone numbers in IRC!
+
+### Frequency of updates
+
+For outages, status should be posted every 30 minutes *or less* in the incident channel.
+
+## Incident timeline
+
+To help write an incident report, it's very useful to include a `TL:` prefix on IRC messages when logging important events and decisions.
```
Comment (bensternthal) [inline docs/incident_response.md]: 👍

**Comment by escattone** [comment]
This looks good so far. The only question that arose for me was the same one that @bensternthal noted above, namely that it wasn't clear which IRC channel should be used for the initial incident announcement, or perhaps we simply choose the IRC channel based on the product, so `#mdndev` for MDN?

File: docs/incident_response.md (hunk 1 above)
Comment (metadave) [reply docs/incident_response.md]: I was thinking that a message should be posted in the primary channel where backend engineers idle for each app:
- #mdndev
- #www
- #basket
- #snippets
- #sumo
etc.

**Comment by metadave** [comment]
This doc was based on things I remember working well from my time at [Chef](https://www.chef.io/). We definitely don't have to adopt this process if it's too heavy etc. How do folks feel?

**Comment by bensternthal** [comment]
I support this. Does not feel to heavy to me.

**Comment by safwanrahman** [comment]
There should be some instruction about when to knock the developer.
Moreover, it would be better if all the components developer names are mentioned.

File: docs/incident_response.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+    - the moc maintains an incident respone guide [here](https://mana.mozilla.org/wiki/display/SECURITY/Incident+Response#IncidentResponse-IncidentResponseTemplate).
+
+Example:
+
+```
+metadave>  there is an MDN outage, we are working in #mdndev       12:05 PM
+metadave>  I'll post an update here within the next 30 minutes     12:05 PM
+```
+
+- Notify team lead for the product impacted.
```
Comment (safwanrahman) [inline docs/incident_response.md]: Team leader means product manager or dev lead?

**Comment by safwanrahman** [comment]
For a longer downtime, a bug should be filled describing the issue and ask `moc` to update the status page with the bug.

**Comment by metadave** [comment]
> There should be some instruction about when to knock the developer.
> Moreover, it would be better if all the components developer names are mentioned.

@safwanrahman that is beyond the scope of this document, see the second line. Contact information is maintained in our internal wiki.

File: docs/incident_response.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+- a Vidyo chat room if needed
+- finally, decide if you need to notify the appropriate engineers that can help resolve the issue
+    - TODO: link to on-call process
+
+Our primary service channels are:
+- `#mdndev`
+- `#www`
+- `#basket`
+- `#snippets`
+- `#sumo`
```
Comment (safwanrahman) [inline docs/incident_response.md]: I think it should be `#sumodev`

File: docs/incident_response.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
+# MozMEAO incident response guide
+
+Stay cool, take a deep breath. We'll solve this problem together, and this guide is here to help.
+
+This document does not describe the on-call or escalation process, but rather the process we use once an issue has been identified.
+
+## Declaring an incident and the incident leader (IL) role
+
+The incident leader (IL) is a Mozillian on staff that serves as the primary point of contact and coordinator for an incident. This person may have been called by the moc, or just noticed that a service is broken.
```
Comment (safwanrahman) [inline docs/incident_response.md]: Mozillian or Staff?

Comment (metadave) [reply docs/incident_response.md]: Mozilla staff only, fixed.

**Comment by escattone** [comment]
@metadave Thanks for this! I do not feel it's too heavy. I think it's worth adopting and we can adjust as we see fit.

**Pull request merged**